File: source/code/python/configuration/redis_cache.py
from stormpath.cache.redis_store import RedisStore
from stormpath.client import Client

client = Client(id='xxx', secret='xxx', cache_options={
    'store': RedisStore,
    'store_opts': {
        'host': 'localhost',
        'port': 6739,
    },
    'ttl': 300,
    'tti': 300,
})

File: source/code/ruby/configuration/redis_cache.rb
client = Stormpath::Client.new(
  api_key_file_location: api_key_file_location,
  cache: {
    store: Stormpath::Cache::RedisStore,
    regions: {
      directories: {
        ttl_seconds: 40,
        tti_seconds: 20
      },
      groups: {
        ttl_seconds: 80,
        tti_seconds: 40,
        store: Stormpath::Cache::MemoryStore
      }
    }
  }
)
